### PR TITLE
Foundation 6 Equalizer

### PR DESCRIPTION
**Pull request by jessedoyle**
:art: :elephant:
* Add a foundation_equalizer_reflow coffeescript listener that
  looks for items with a data-equalizer-reflow attribute and
  triggers a Foundation reInit of equalizer on the click event.
* Remove commented header_menu code.
* Update version to 4.0.0.pre1 - a major version change along
  with Foundation 6.

### CODE REVIEW

File: lib/ama_layout/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module AmaLayout
-  VERSION = '3.2.0'
+  VERSION = '4.0.0.pre1'
```
Comment (kaytcampbell) [inline lib/ama_layout/version.rb]: interesting, I didn't know you could do that!

Comment (jessedoyle) [inline lib/ama_layout/version.rb]: Yea - it really helps bundler tell when a gem has changed. You could even do `4.0.0.kayt` if you wanted.

**Comment by jessedoyle** [comment]
:wave:

**Comment by rubene** [comment]
🐘 🐘 🐘

**Comment by kaytcampbell** [comment]
:art: :art: :art:

**Pull request merged**